File: core/src/main/java/org/biacode/spring/netty/core/annotation/NettyCommand.java
package org.biacode.spring.netty.core.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by Arthur Asatryan.
 * Date: 1/31/18
 * Time: 1:47 PM
 */
@Target(ElementType.METHOD)
@Retention(RetentionPolicy.RUNTIME)
public @interface NettyCommand {
    String value();
}

File: src/main/kotlin/org/biacode/hermes/spring/netty/config/annotation/NettyCommand.kt
package org.biacode.hermes.spring.netty.config.annotation

/**
 * Created by Arthur Asatryan.
 * Date: 1/25/18
 * Time: 3:46 PM
 */
@Target(AnnotationTarget.FUNCTION)
@Retention(AnnotationRetention.RUNTIME)
annotation class NettyCommand(val value: String)